### DISCUSSION TITLE
`create-next-app` should let the user retry if the import alias doesn't match regex

### DISCUSSION THREAD

**Discussion by c-ehrlich** [vercel/next.js#45132] [Ideas]
It's easy to accidentally type the wrong thing when asked for the import alias (for example see a screenshot below where I typed `~` instead of `~/*`. At this point the CLI exits and you have to start over.

<img width="722" alt="Screenshot 2023-01-21 at 11 44 11" src="https://user-images.githubusercontent.com/8353666/213882946-eb63fc5c-f788-4c69-8a70-73b94a2ce320.png">

I believe it would be a nicer experience if the CLI just told you that the input doesn't match the expected format, and then gave you another chance to enter it, as shown below.

<img width="742" alt="Screenshot 2023-01-21 at 20 01 49" src="https://user-images.githubusercontent.com/8353666/213882949-63fd7f1d-5691-4bc0-a98b-3d82a556cf64.png">

I could implement this if it seems like a good idea.

**Comment by Nsttt** [comment]
100% agree.

**Reply by c-ehrlich** [reply]
Shout outs to the person who sniped the PR lol
